.env.example

```
@@ -0,0 +1,3 @@
+DB_URL="link"
+PORT = "number"
+JWT_SECRET="secretCode"
```

```suggestion
DB_URL=
PORT=
JWT_SECRET=secretCode
```
Tak zapisanie wystarczy.